src/SonarScanner.MSBuild.TFS/Resources.resx

```diff
@@ -165,4 +183,7 @@
   <data name="TRX_WARN_InvalidTrx" xml:space="preserve">
     <value>Located trx file is not a valid xml file. File: {0}. File load error: {1}</value>
   </data>
+  <data name="WARN_XamlBuildDeprecated" xml:space="preserve">
+    <value>** WARNING: Support for XAML builds is deprecated since version 4.1 and will be removed in version 5.0 of the Scanner for MSBuild **</value>
```

Comment:
This message is still being emitted by the SummaryReportBuilder, but from the GitHub milestone it looks like the new version will be v5.0.

If that's correct, I'm not sure what the correct thing to do is - drop the message, drop the functionality, or change the message.

Reply:
Yeah that's what i figured out recently, let's change the message first, and drop the code in a future release (ie 5.1)